src/types/ErrorMessages.ts

```diff
@@ -1,5 +1,7 @@
 enum ErrorMessages {
-  QualifierCode = "This code could not be recognised, contact the courts to verify the correct information and report to the Bichard 7 team. Resolve exception by manually updating the PNC with correct code."
+  QualifierCode = "This code could not be recognised, contact the courts to verify the correct information and report to the Bichard 7 team. Resolve exception by manually updating the PNC with correct code.",
+  HO100251_error_prompt = "This code is not valid, contact the courts to correct offence code.",
```

Comment:
This name would be better camel cased to `HO100251ErrorPrompt` to be consistent with `QualifierCode`, and the same with `HO100306_error_prompt`